indexer/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@filecoin-station/spark-piece-indexer",
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "dry-run": "node bin/dry-run.js",
+    "migrate": "node bin/migrate.js",
+    "start": "node bin/piece-indexer.js",
+    "lint": "standard",
+    "test": "node --test"
```

Comment:
Could we include `lint` in `test`? To me, when I run `npm test`, I expect it to tell me whether CI will pass. This is not the case here. Or do you have a reason for this?

Reply:
That makes sense.

However, your proposed change would mean that we execute the linter twice when you run `npm test` in the monorepo root.

I am using the setup we already have in other places:
- https://github.com/filecoin-station/spark-stats/blob/28fbe3a9b10c907f11052651d0f939f286397eec/observer/package.json#L9-L10
- https://github.com/filecoin-station/spark-api/blob/0d79f8b060aac01d2f8c04d622aed3aded3dc707/api/package.json#L11-L12

What you can do in the monorepo root

- `npm test` to run all CI checks in all workspaces (directories)
- `npm test -w indexer` to run only the tests only in the `indexer` directory
- `npm lint -w indexer` to run only the linters only in the `indexer` directory